zb-db/src/main/java/io/camunda/zeebe/db/impl/rocksdb/transaction/TransactionalColumnFamily.java

```diff
@@ -187,6 +225,30 @@ public boolean isEmpty() {
     return isEmpty.get();
   }
 
+  private void assertDoesNotExist(final ZeebeTransaction transaction) throws Exception {
+    final var value =
+        transaction.get(
```

Comment:
:thinking: I still wonder whether the good would be simpler if the signature was `get(transactionDb, columnFamilyContext`

But maybe that is something for another day